h2o-core/src/main/java/water/UDPHeartbeat.java

```diff
@@ -17,12 +17,6 @@
         return ab;
       }
       Paxos.doHeartbeat(ab._h2o);
-
-      // record clients this node has ever heard off only in multicast mode.
```

Comment:
Please explain why is this not needed anymore.

Reply:
The code is still needed, but I just moved to Paxos so it's close to the rest of methods taking care of clients/clouding